Content.Server/Chat/Systems/ChatSystem.Radio.cs

```diff
@@ -45,6 +71,25 @@ private void ShutdownRadio()
         // TODO: Turn common into a true frequency and support multiple aliases.
         var isRadioMessage = false;
         RadioChannelPrototype? channel = null;
+
+        // Replaces :h with whatevs the headset's "default" is
+        var locChannelMessage = message.StartsWith(":h") || message.StartsWith(".h");
+        if (locChannelMessage && message.Length >= 2)
+        {
+            if (_inventory.TryGetSlotEntity(source, "ears", out var headsetUid) && HasComp<HeadsetComponent>(headsetUid))
+            {
+                message = message[2..].TrimStart();
+                var headsetUid2 = headsetUid ?? default(EntityUid);
+                var entityPrototype = Prototype(headsetUid2) ?? default(EntityPrototype);
+                if (entityPrototype != null)
+                {
+                    var entityId = entityPrototype.ID;
+                    message = headsetCommonDict[entityId] + message;
+                }
```

Comment:
Entity prototype IDs should only be used for spawning and nothing else. The reason for this is it breaks with inheritance.

Reply:
No longer grabbing the Prototype and getting its ID